src/nginx/azext_nginx/_help.py

```diff
@@ -0,0 +1,234 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['nginx deployment list'] = """
+    type: command
+    short-summary: "List all Nginx Deployments under the specified resource group; List all deployments under the specified subscription."
+    examples:
+      - name: Deployments_ListByResourceGroup
+        text: |-
+               az nginx deployment list --resource-group myResourceGroup
+      - name: Deployments_List
+        text: |-
+               az nginx deployment list
+"""
```

Comment:
Because you use aaz-dev-tools to generate atomic commands. So you should add examples and summaries in aaz-dev-tools. When you generate code, that information will be contained in the Command class description. Instead of writing them in _help.py file.

Reply:
> Please create a PR to submit the atomic command models in [aaz](https://github.com/Azure/aaz) as well.

Created PR https://github.com/Azure/aaz/pull/24

Reply:
> Because you use aaz-dev-tools to generate atomic commands. You should add examples and summaries in aaz-dev-tools. When you generate code, that information will be contained in the Command class description. Instead of writing them in _help.py file.

Oh. Thanks for this. Done